frontend/src/services/eventHandlers/gameHandler.js

```diff
@@ -8,12 +8,18 @@ import {
 } from "@/modules/scribble/stores/states/gamestates";
 import { store } from "@/store";
 
+const NANOSECOND_TO_SECONDS_FACTOR = 1000000000;
+
 // Event handler for Game API
 export default class GameHandler {
     constructor() {
         this.setGameStateKey = "scribble/setGameState";
     }
 
+    _convertToSeconds(durationNS) {
```

Comment:
for private methods we should be using #, outlined here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields#:~:text=Private%20fields%20are%20accessible%20on,only%20static%20methods%20still%20holds.&text=Private%20static%20fields%20are%20added,constructor%20at%20class%20evaluation%20time.

Reply:
Actually, it looks like its experimental and not supported in firefox. Not sure now if we should do this, what do you think?

Reply:
I think if its not supported we shouldn't use it, as we do use firefox for testng